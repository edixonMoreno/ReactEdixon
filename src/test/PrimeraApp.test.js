import React from 'react'
//import {toBeInTheDocument} from "@testing-library/jest-dom";
import PrimeraApp from "../PrimeraApp";
import { shallow } from 'enzyme';


describe('Pruebas de >PrimeraApp />', () => {
    
//     test('debe mostrar el mensaje "Hola soy Goku" ', () => {
        
//         const saludo = "Hola Soy Goku";

//         const {getByText} = render( <PrimeraApp saludo={ saludo } /> );

//         expect( getByText(saludo) ).toBeInTheDocument();
//     })
  
    test('debe mostrar <PrimeraApp /> correctamente ', () => {
        
        const saludo = "Hola Soy Edixon";
        const wrapper = shallow( <PrimeraApp saludo = {saludo} />)

        expect( wrapper ).toMatchSnapshot();
    })
    
    test('debe mostrar el subtitulo enviado por props', () => {
        
        const saludo = "Hola Soy Edixon";
        const subTitulo = "Soy el subtitulo";
        const wrapper = shallow( <PrimeraApp saludo={saludo} subtitulo={subTitulo} />);

        const textoParrafo = wrapper.find("p").text();
       
        expect(textoParrafo).toBe(subTitulo);
    })
    

 })
