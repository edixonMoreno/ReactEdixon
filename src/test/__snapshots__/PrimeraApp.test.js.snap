// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de >PrimeraApp /> debe mostrar <PrimeraApp /> correctamente  1`] = `
<Fragment>
  <h1>
    Hola Soy Edixon
    !!
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
